Bizyhood/Core.php

```diff
@@ -1945,6 +1959,73 @@ function bizyhood_plugin_action_links( $links ) {
        return $links;
     }
     
+    
+    function load_rss() {
```

Comment:
Spyros, I'm now getting the following error:

```
This page contains the following errors:

error on line 9 at column 73: Entity 'raquo' not defined
Below is a rendering of the page up to the first error.
```

I'm testing by making the API key invalid.